packages/lesswrong/lib/collections/moderatorActions/helpers.ts

```diff
@@ -97,9 +97,10 @@ export function getCurrentContentCount(user: UserContentCountPartial) {
   return postCount + commentCount
 }
 
-export function getReasonForReview(user: DbUser | SunshineUsersList, override?: true) {
-  if (override) return 'override';
-
+export function getReasonForReview(user: DbUser|SunshineUsersList):
+  {needsReview: false, reason: "alreadyApproved"|"noReview"}
```

Comment:
I'd feel _so_ much better about this being extracted to a named type 😅  (I expect it to be pretty awkward to read inline in most editor/screen combinations)